spec/spec_helper.rb

```diff
@@ -39,19 +43,20 @@
 
 ShowMeTheCookies.register_adapter(ENV['browser'].to_sym, ShowMeTheCookies::Selenium)
 
-Capybara.app_host = 'https://spree-qa-lynn.herokuapp.com'
+Capybara.app_host = 'https://lynn-spree.herokuapp.com'
 
 Capybara.run_server = false
-Capybara.page.driver.browser.manage.window.maximize
 
 RSpec.configure do |config|
   config.include Capybara::DSL
   config.include MainHelper
   config.include LinksHelper
+  config.include PDPHelper
   config.include ShowMeTheCookies
 
   config.after do
-    Capybara.reset_sessions!
+    logout
```

Comment:
It's not needed when you call `Capybara.reset_session!` afterwards.

Reply:
it was changed in different PR, might have even been merged to master already (: 
or will be merged with the 7th PR